packages/manager/apps/web/client/app/hosting/cdn/flush/hosting-cdn-flush.routing.js

```diff
@@ -0,0 +1,27 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.dashboard.cdn.flush', {
+    url: '/flush',
+    layout: 'modal',
+    params: {
+      cdnProperties: null,
+      domain: null,
+      productId: null,
```

Comment:
Could `cdnProperties` be retrieved for those params ? That way they could appear in URL